lib/benchee/utility/parallel.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Benchee.Utility.Parallel do
+  @moduledoc false
+
+  @doc """
+  A utility function for mapping over an enumerable collection in parallel.
+  """
+  @spec map(Enum.t, fun) :: list
+  def map(enumerable, func) do
+    enumerable
+    |> Enum.map(fn(element) -> fn() -> func.(element) end end)
+    |> Enum.map(fn(func) -> Task.async(func) end)
```

Comment:
feels like one map too many imo, but that might be style. Souldn't:

```
|> Enum.map(fn(element) -> Task.async(fn -> func.(element)) end end)
```

do the same with about the same complexity?